TaskWithInheritanceAndPolymorphism/TaskWithInheritanceAndPolymorphism.cpp

```diff
@@ -0,0 +1,139 @@
+﻿#include <iostream>
+#include <memory>
+#include <vector>
+
+class Figure
+{
+public:
+	Figure()
+	{
+		std::cout << "Figure()" << std::endl;
+	}
+	~Figure()
+	{
+		std::cout << "~Figure()" << std::endl;
+	}
+
+	virtual const void what() = 0;
```

Comment:
"const void" seriously? Please move the const specifier to the end of the function declaration like this:
`virtual void what() const = 0;`

The function will become a const function which means you can call it for constant objects as well.

 Please apply this comment to other functions.

Reply:
Fixed